libs/cpp.json

```diff
@@ -1017,5 +1017,9 @@
   {"id": "cpp.yomm2", "name": "YOMM2", "tech": ["app-arch"], "tags": ["multiple-dispatch"], "repo": "jll63/yomm2",
   "imports": ["yorel/"], "examples": ["https://github.com/jll63/yomm2/tree/master/examples"], "status": "model-ready"},
   {"id": "cpp.stlport", "name": "STLport", "tech": ["app-arch"], "tags": ["cross-platform"], "repo": "https://sourceforge.net/p/stlport/code/ci/master/tree/",
-  "imports": [], "examples": ["https://sourceforge.net/p/stlport/code/ci/master/tree/test/"], "status": "awaiting-model"}
+  "imports": [], "examples": ["https://sourceforge.net/p/stlport/code/ci/master/tree/test/"], "status": "awaiting-model"},
+  {"id":"cpp.librdkafka","imports":["librdkafka/rdkafkacpp.h"],"name":"librdkafka","repo":"edenhill/librdkafka","tags":["streaming"],"tech":["app-arch"],"status":"awaiting-model"},
+  {"id":"cs.azure-c-shared-utility","imports":["azure_c_shared_utility/"],"name":"Microsoft.Azure.C.SharedUtility","repo":"Azure/azure-c-shared-utility","tags":["toolkit"],"tech":["utils"],"status":"awaiting-model"},
```

Comment:
this is definitely cloud. this seems similar to py.boto

Reply:
are you sure, the description says "C library providing common functionality for basic tasks (like string, list manipulation, IO, etc.)."